torch/ao/quantization/quantize_pt2e.py

```diff
@@ -103,6 +103,7 @@ def calibrate(model, data_loader):
     # to be quantized before fusion
     # TODO: (maybe) rewrite this with subgraph_rewriter
     _fuse_conv_bn_(model)
+    quantizer.transform_for_annotation(model)
```

Comment:
Should we use the same method for  QAT fusion as well? Or is it intended for something else

Reply:
this is not for QAT fusion, plan for QAT transforms is still to put these in the flow for now